examples/hosts/app-integration/external-data/src/mock-service/service.ts

```diff
@@ -115,13 +115,13 @@ export async function initializeCustomerService(props: ServiceProps): Promise<Se
         externalDataSource.fetchData().then(
             (data) => {
                 console.log(`SERVICE: Returning current task list:\n"${data}".`);
-                result.send({taskList: data});
+                result.send({ taskList: JSON.parse(data.body.toString()) as object });
```

Comment:
Is `as object` needed?

Reply:
Yes I believe so because JSON.parse returns any type which Typescript doesn't like